static/app/components/modals/inviteMembersModal/useInviteModal.tsx

```diff
@@ -131,11 +133,13 @@ export default function useInviteModal({organization, initialData, source}: Prop
               ? errorResponse.email[0]
               : errorResponse.email;
 
-        const error = emailError || t('Could not invite user');
+        const orgLevelError = errorResponse?.organization || null;
```

Comment:
i could have made this 
```suggestion
        const orgLevelError = errorResponse?.organization || undefined;
```

which could have helped me do
```suggestion
        error: orgLevelError;
```
down below

Reply:
nit: you can simplify this to: `const orgLevelError = errorResponse?.organization` -- as this will be set to `undefined` if there isn't an error response or organization error. Doing that also helps the typing a little since it's using [`undefined`](https://github.com/getsentry/sentry/pull/66483/files#diff-4a8cbb7e3d395c114db1c36d4bb92ff5ffd4640e2f3013c21cfc359e95a7b30cR94) already?

Reply:
ohhh, let me do that then, thank you!

Reply:
done!